apps/condo/domains/ticket/tasks/sendTicketChangedNotifications.js

```diff
@@ -0,0 +1,252 @@
+const dayjs = require('dayjs')
+const get = require('lodash/get')
+const { v4: uuid } = require('uuid')
+
+const conf = require('@open-condo/config')
+const { featureToggleManager } = require('@open-condo/featureflags/featureToggleManager')
+const { getLogger } = require('@open-condo/keystone/logging')
+const { getById, getByCondition, getSchemaCtx } = require('@open-condo/keystone/schema')
+
+const { COUNTRIES } = require('@condo/domains/common/constants/countries')
+const { SEND_TELEGRAM_NOTIFICATIONS, SMS_AFTER_TICKET_CREATION } = require('@condo/domains/common/constants/featureflags')
+const { TWO_OR_MORE_SPACES_REGEXP } = require('@condo/domains/common/constants/regexps')
+const { md5 } = require('@condo/domains/common/utils/crypto')
+const { TICKET_ASSIGNEE_CONNECTED_TYPE, TICKET_EXECUTOR_CONNECTED_TYPE,
+    TICKET_STATUS_RETURNED_TYPE,
+    TICKET_STATUS_OPENED_TYPE,
+    TICKET_STATUS_IN_PROGRESS_TYPE,
+    TICKET_STATUS_COMPLETED_TYPE,
+    TICKET_STATUS_DECLINED_TYPE, TRACK_TICKET_IN_DOMA_APP_TYPE,
+} = require('@condo/domains/notification/constants/constants')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { ORGANIZATION_NAME_PREFIX_AND_QUOTES_REGEXP } = require('@condo/domains/organization/constants/common')
+const { Resident } = require('@condo/domains/resident/utils/serverSchema')
+const { STATUS_IDS } = require('@condo/domains/ticket/constants/statusTransitions')
+const { detectTicketEventTypes, TICKET_CREATED, ASSIGNEE_CONNECTED_EVENT_TYPE, EXECUTOR_CONNECTED_EVENT_TYPE,
+    STATUS_CHANGED_EVENT_TYPE, TICKET_WITHOUT_RESIDENT_CREATED_EVENT_TYPE,
+} = require('@condo/domains/ticket/utils/detectTicketEventTypes')
+
+const { sendTicketCreatedNotifications } = require('./index')
+
+
+const appLogger = getLogger('condo')
+const taskLogger = appLogger.child({ module: 'tasks/sendTicketChangedNotifications' })
```

Comment:
Which name it will be?
Why not just `const logger = getLogger('tasks/sendTicketChangedNotifications')` ?

Reply:
Honestly, I don't see any fundamental difference. 
I can write it in the name :)

Reply:
I left only the name